previousnext/k8s-skpr-solr
# Add CircleCI build spec.

#### What does this PR do?

Adds a CircleCI build spec which currently just runs unit tests.

#### How should this be manually tested?

Check out the CircleCI build log for this branch - https://circleci.com/gh/previousnext/k8s-skpr-solr/5
